storm-client/test/jvm/org/apache/storm/TestConfigValidate.java

```diff
@@ -84,21 +83,6 @@ public void invalidConfigTest() throws InvocationTargetException, NoSuchMethodEx
         ConfigValidation.validateFields(conf);
     }
 
-    @Test(expected = InvalidTopologyException.class)
```

Comment:
Yes please do it if you really don't mind. I think we need to have broader coverage of tests.